qlfile

```diff
@@ -44,10 +44,10 @@ github cl-gearman svetlyak40wt/cl-gearman :branch patches
 # https://github.com/thephoeron/cl-isaac/pull/13
 github cl-isaac svetlyak40wt/cl-isaac :branch fix-value-1-error
 
+# required by fukamachi/qlot as a dependency
+ql dexador
 
-# Until we solve https://github.com/fukamachi/qlot/issues/265
-# and https://github.com/ultralisp/ultralisp/issues/278
-# github qlot fukamachi/qlot :ref a4abc8921c4a8cf1c43f453bb2106ee56ae3c13d
+github qlot fukamachi/qlot
```

Comment:
Changes in this file should note be necessary.

Try to remove them and run `qlot update` instead, then commit a new version of qlfile.lock.

Reply:
OK, read it a bit too quickly, removing all my changes and re-running `lot update`...

Reply:
Back to this error `Unexpected error: Component "qlot" not found`, which set me on this path of tinkering with `qlfile` in the first place...